### PR TITLE
Add `glasses-`[`aviator`/`rectangle`/`cat eye`] icons and rename past glasses icon

### PR DESCRIPTION
**Pull request by hibatillah**
### Add new types glasses
- `glasses-aviator`
- `glasses-rectangle`
- `glasses-cateye`

### Rename past glasses icon
from `glasses` to `glasses-round`, because of the use of the type for glasses

### CODE REVIEW

**Comment by hibatillah** [comment]
there's problem with the icons, i dont know how to fix it. pls help!

File: icons/glasses-round.json
Comment (karsa-mistmere) [inline icons/glasses-round.json:1]: If we rename this, we would need to add an alias to this JSON file for backwards compatibility.

File: icons/glasses-rectangle.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 14C22 13.4477 21.5523 13 21 13H15C14.4477 13 14 13.4477 14 14V17C14 17.5523 14.4477 18 15 18H21C21.5523 18 22 17.5523 22 17V14Z" stroke="black" stroke-width="2"/>
+  <path d="M2 14C2 13.4477 2.44772 13 3 13H9C9.55228 13 10 13.4477 10 14V17C10 17.5523 9.55228 18 9 18H3C2.44772 18 2 17.5523 2 17V14Z" stroke="black" stroke-width="2"/>
+  <line x1="10" y1="15" x2="14" y2="15" stroke="black" stroke-width="2"/>
+  <path d="M21.5 13L19 7C18.3 5.7 17.5 5 16 5" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+  <path d="M2.5 13L5 7C5.7 5.7 6.4 5 8 5" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (karsa-mistmere) [inline icons/glasses-rectangle.svg:16]: Hi @hibatillah,

When contributing, it's best to allow the pre commit hooks to run, for that to happen, you need to run `pnpm install`.
Alternatively, you can use either [SVGOMG](https://jakearchibald.github.io/svgomg/) to optimize your icon and then @jguddas's awesome (although admittedly still in beta) edit tool: https://github.com/lucide-icons/lucide/pull/1022

What you should end up with is something like this (i.e. no explicit stroke or fill attributes on any path elements):
```suggestion
  <rect x="14" y="13" width="8" height="5" rx="1" />
  <rect x="2" y="13" width="8" height="5" rx="1" />
  <path d="M10 15h4" />
  <path d="M21.5 13 19 7c-.7-1.3-1.5-2-3-2" />
  <path d="M2.5 13 5 7c.7-1.3 1.4-2 3-2" />
```

File: icons/glasses-aviator.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M2 13C2 12.4477 2.44772 12 3 12H9C9.55228 12 10 12.4477 10 13V15C10 17.2091 8.20914 19 6 19H5C3.34315 19 2 17.6569 2 16V13Z" stroke="black" stroke-width="2"/>
+  <path d="M22 13C22 12.4477 21.5523 12 21 12H15C14.4477 12 14 12.4477 14 13V15C14 17.2091 15.7909 19 18 19H19C20.6569 19 22 17.6569 22 16V13Z" stroke="black" stroke-width="2"/>
+  <line x1="10" y1="15" x2="14" y2="15" stroke="black" stroke-width="2"/>
+  <path d="M21 12L18.5 6C17.8 4.7 17 4 15.5 4" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+  <path d="M3 12L5.5 6C6.2 4.7 6.9 4 8.5 4" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (karsa-mistmere) [inline icons/glasses-aviator.svg:16]:
```suggestion
  <path d="M10 15h4" />
  <path d="M2 13a1 1 0 0 1 1-1h6a1 1 0 0 1 1 1v2a4 4 0 0 1-4 4H5a3 3 0 0 1-3-3Z" />
  <path d="M22 13a1 1 0 0 0-1-1h-6a1 1 0 0 0-1 1v2a4 4 0 0 0 4 4h1a3 3 0 0 0 3-3Z" />
  <path d="m21 12-2.5-6c-.7-1.3-1.5-2-3-2" />
  <path d="m3 12 2.5-6c.7-1.3 1.4-2 3-2" />
```

File: icons/glasses-cateye.svg [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M2 13H8C9.10457 13 10 13.8954 10 15V18C10 18.5523 9.55228 19 9 19H5C3.34315 19 2 17.6569 2 16V13Z" stroke="black" stroke-width="2"/>
+  <path d="M22 13H16C14.8954 13 14 13.8954 14 15V18C14 18.5523 14.4477 19 15 19H19C20.6569 19 22 17.6569 22 16V13Z" stroke="black" stroke-width="2"/>
+  <line x1="10" y1="15" x2="14" y2="15" stroke="black" stroke-width="2"/>
+  <path d="M21.5 13L19 7C18.3 5.7 17.5 5 16 5" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+  <path d="M2.5 13L5 7C5.7 5.7 6.4 5 8 5" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (karsa-mistmere) [inline icons/glasses-cateye.svg:16]:
```suggestion
  <path d="M10 15h4" />
  <path d="M2 13h6a2 2 0 0 1 2 2v3a1 1 0 0 1-1 1H5a3 3 0 0 1-3-3Z" />
  <path d="M22 13h-6a2 2 0 0 0-2 2v3a1 1 0 0 0 1 1h4a3 3 0 0 0 3-3Z" />
  <path d="M21.5 13 19 7c-.7-1.3-1.5-2-3-2" />
  <path d="M2.5 13 5 7c.7-1.3 1.4-2 3-2" />
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Comment by jguddas** [comment]
Not to be that guy but there needs to be a description of the uniq use case for each of those icons.

**Pull request closed without merging**